<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiAjoutActeurArticle</class>
 <widget class="QWidget" name="GuiAjoutActeurArticle">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>358</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Gestion des acteurs par article</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../SeboApp.qrc">
    <normaloff>:/SeboApp/Resources/favicon.png</normaloff>:/SeboApp/Resources/favicon.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblActeursDispos">
       <property name="text">
        <string>Acteurs disponibles :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lblFiltrerRole">
         <property name="text">
          <string>Filtrer par rôles :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbFiltrerRoles">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnToutAfficher">
         <property name="minimumSize">
          <size>
           <width>27</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>27</width>
           <height>27</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../SeboApp.qrc">
           <normaloff>:/SeboApp/Resources/toutAfficher.png</normaloff>:/SeboApp/Resources/toutAfficher.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tvActeursDispos">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnGererActeurs">
         <property name="text">
          <string>Gérer les acteurs</string>
         </property>
         <property name="icon">
          <iconset resource="../../SeboApp.qrc">
           <normaloff>:/SeboApp/Resources/modifier2.png</normaloff>:/SeboApp/Resources/modifier2.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGererRoles">
         <property name="text">
          <string>Gérer les rôles</string>
         </property>
         <property name="icon">
          <iconset resource="../../SeboApp.qrc">
           <normaloff>:/SeboApp/Resources/modifier2.png</normaloff>:/SeboApp/Resources/modifier2.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="btnAjouter">
       <property name="text">
        <string>----&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRetirer">
       <property name="text">
        <string>&lt;----</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lblActeursLies">
       <property name="text">
        <string>Acteurs liés à l'article :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tvActeursLies">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnValider">
         <property name="text">
          <string>Valider</string>
         </property>
         <property name="icon">
          <iconset resource="../../SeboApp.qrc">
           <normaloff>:/SeboApp/Resources/valider.png</normaloff>:/SeboApp/Resources/valider.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAnnuler">
         <property name="text">
          <string>Annuler</string>
         </property>
         <property name="icon">
          <iconset resource="../../SeboApp.qrc">
           <normaloff>:/SeboApp/Resources/Annuler.png</normaloff>:/SeboApp/Resources/Annuler.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../SeboApp.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnAnnuler</sender>
   <signal>clicked()</signal>
   <receiver>GuiAjoutActeurArticle</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>702</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
